services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/dao/SubnetRepository.java

```diff
@@ -33,21 +33,21 @@
 
 @Repository
 @ComponentScan(value="com.futurewei.alcor.common.db")
-public class SubnetRepository implements ICacheRepository<SubnetState> {
+public class SubnetRepository implements ICacheRepository<SubnetWebObject> {
 
     private static final Logger logger = LoggerFactory.getLogger();
 
     private static final String KEY = "SubnetState";
 
-    public ICache<String, SubnetState> getCache() {
+    public ICache<String, SubnetWebObject> getCache() {
```

Comment:
@kevin-zhonghao Do we want to split it to web object and db object for subnet?

Reply:
I think we could just make one object for subnet ~ but we can talk about it